bfcfg

```diff
@@ -68,6 +68,11 @@ sb_custom_mode_sysfs=${efivars}/BfCfgSbCustomMode-${efi_bfcfg_var_guid}
 pass_settings_sysfs=${efivars}/BfCfgPassSettings-${efi_bfcfg_var_guid}
 bf_modes_sysfs=${efivars}/BfCfgBfModes-${efi_bfcfg_var_guid}
 boot_wdog_sysfs=${efivars}/BfBootWdog-120b4e59-904a-4fe2-9e03-54e9a29ffb22
+bf_profile_root_key=${efivars}/BfProfileRootKey-f4d4a988-e8c5-11ef-8afc-0242c0a80103
+bf_profile_ca_cert_nvidia=${efivars}/BfProfileCaCertNvidia-f4d4a988-e8c5-11ef-8afc-0242c0a80103
+bf_profile_ca_cert_oem=${efivars}/BfProfileCaCertOem-f4d4a988-e8c5-11ef-8afc-0242c0a80103
+bf_profile_config_ca_cert_nvidia=${efivars}/BfProfileConfigSubCaCertNvidia-f4d4a988-e8c5-11ef-8afc-0242c0a80103
+bf_profile_config_ca_cert_oem=${efivars}/BfProfileConfigSubCaCertOem-f4d4a988-e8c5-11ef-8afc-0242c0a80103
```

Comment:
Should we rename as `bf_profile_config_sub_ca_cert_nvidia` and `bf_profile_config_sub_ca_cert_oem` just to maintain naming coherency ?

Reply:
Done